Explorer/Assets/Scripts/ECS/Unity/Textures/Components/VideoTextureComponent.cs

```diff
@@ -0,0 +1,24 @@
+﻿using DCL.Optimization.Pools;
+using System;
+using UnityEngine;
+
+namespace ECS.Unity.Textures.Components
+{
+    public struct VideoTextureComponent : IDisposable
+    {
+        public Texture2D Texture { get; private set; }
+        private readonly IExtendedObjectPool<Texture2D> videoTexturePool;
+
+        public VideoTextureComponent(IExtendedObjectPool<Texture2D> videoTexturePool)
+        {
+            this.videoTexturePool = videoTexturePool;
+            Texture = this.videoTexturePool.Get();
+        }
+
+        public void Dispose()
```

Comment:
1. You never call it
2. Don't add unnecessary responsibilities to the component: just pass the pool as an argument to `Dispose` from systems, don't pass it to the `ctor` either

Reply:
as for 1., it is actually called in the `CleanUpVideoPlayerSystem`, I will move the pool to it then.